com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -203,6 +203,8 @@ In the Decal shader, the property `_MetallicRemapMin` have been added, and `_Met
 
 From 10.x, a new pass ScenePickingPass have been added to all the shader and master node to allow the editor to correctly handle the picking with tesselated objects and backfaced objects.
 
+From 10.x, if material ambient occlusion needs to be applied to probe volume GI and the material is deferred, the material needs to define `HAS_PAYLOAD_WITH_UNINIT_GI` constant and a function `float GetUninitializedGIPayload(SurfaceData surfaceData)` that returns the AO factor that is desired to be applied. No action is needed for forward only materials or if no material AO needs to be applied to probe volume GI.
```

Comment:
you can remove this statement, 
1) it is not the right location (upagre 2020.1 to 2020.2)
2) we haven't ship the feature, so we don't care.

Reply:
I now understand why it is there, should be in 2021.1 to 2021.2 upgrading guide (that we haven't created)